### ISSUE TITLE
Yolov3 got fail_accuracy

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#503]
### 🐛 Describe the bug

torchbench_float16_training
xpu  train yolov3                             
W0626 13:18:03.033000 139736640825152 torch/_inductor/utils.py:1221] [3/0_2] DeviceCopy in input program
W0626 13:18:03.033000 139736640825152 torch/_inductor/utils.py:1221] [3/0_2] DeviceCopy in input program
W0626 13:18:03.212000 139736640825152 torch/_inductor/utils.py:1221] [3/0_2] DeviceCopy in input program
W0626 13:18:03.213000 139736640825152 torch/_inductor/utils.py:1221] [3/0_2] DeviceCopy in input program
W0626 13:18:03.390000 139736640825152 torch/_inductor/utils.py:1221] [3/0_2] DeviceCopy in input program
W0626 13:18:03.391000 139736640825152 torch/_inductor/utils.py:1221] [3/0_2] DeviceCopy in input program
W0626 13:19:19.452000 139736640825152 torch/_dynamo/utils.py:1452] Found nan in reference. Consider running in higher precision.
E0626 13:19:19.452000 139736640825152 torch/_dynamo/utils.py:1478] RMSE (res-fp64): 0.02577, (ref-fp64): nan and shape=torch.Size([4, 3, 12, 16, 85]). res.dtype: torch.float16, multiplier: 2.000000, tol: 0.001000
fail_accuracy

### Versions

torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/31c400195d63064940242220dc9100322d36bac4
pytorch: 0f81473d7b4a1bf09246410712df22541be7caf3 + PRs: 127277,129120
device: PVC 1100, 803.61, 0.5.1

**Comment by retonym** [comment]
No accuracy issue in latest torch xpu ops
